qiskit/dagcircuit/dagcircuit.py

```diff
@@ -325,8 +325,8 @@ def _check_condition(self, name, condition):
         Raises:
             DAGCircuitError: if conditioning on an invalid register
         """
-        # Verify creg exists
-        if condition is not None and condition[0].name not in self.cregs:
+        if (condition is not None and condition[0] not in self.clbits and
+                condition[0].name not in self.cregs):
```

Comment:
Can you update the docstring above to include the possibility `condition` is `(Clbit, int)`?

Reply:
Changes done.